### ISSUE TITLE
Trimming the context

### ISSUE THREAD

**Issue by iherman** [w3c/json-ld-rc#6]
Looking at the current context file with a critical eye, I would consider really trimming it considerably.

The choice of the RDFa default context was motivated by the lack of a `@context`-like mechanism in RDFa, meaning that _all_ RDFa usages presupposed a load of repeated prefix statement (which are also awkward in RDFa). The rules to establish the RDFa context were:

1. we performed a search over the vocabulary usage _at that time_ and chose the top ones
2. we established the rule whereby all vocabularies defined at W3C via a WG or an IG would be automatically added to the list.
3. we never remove anything from the list

Mainly (1) above is of course questionable for everyday usage: `ssn`, `time`, `og`, `snomed` may all be questionable in view of JSON-LD usage. `snomed`, for example, is of a very specific usage (although important in a narrow area), and I am not sure it should be part of a generic thing.

At this moment, my preferred approach would be to greatly reduce the prefixes. I would retain only the following:

```
        "dc": "http://purl.org/dc/elements/1.1/",
        "dc11": "http://purl.org/dc/elements/1.1/",
        "dct": "http://purl.org/dc/terms/",
        "dcterms": "http://purl.org/dc/terms/",
        "dctype": "http://purl.org/dc/dcmitype/",
        "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
        "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
        "schema": "http://schema.org/",
        "xsd": "http://www.w3.org/2001/XMLSchema#",
```

reasons: DC and schema are the only really ubiquitous vocabularies out there, and the rdfs+rdfs+xsd are part of the 'core' anyway. Note that I do not even include 'foaf' and 'owl'; I am not sure 'foaf' is used by anyone except by the die hard semantic web people, and the same holds for OWL.

**Comment by pchampin** [comment]
In my experience, the `dc` prefix is not used consistently (some people use it as `dc11`, others use it as `dcterms`). Following [the Zen of Python](https://www.python.org/dev/peps/pep-0020/) “*in the face of ambiguity, refuse the temptation to guess*”, I would rather not include `dc` at all... 

It is better to get an error than to get something that *seem* to work, but with the wrong IRI. The error can be frustrating, but if that helps making people aware of the ambiguity, I think this is for the best (following the Zen of my Grandpa “*this is for your own good*” :wink:).

**Comment by iherman** [comment]
@pchampin I have sympathy with what you say... Another possibility that we try to contact Tom Baker and ask him for the 'official' standpoint of DCMI.

**Comment by davidlehn** [comment]
How close should this align with the rdfa context?
https://www.w3.org/2011/rdfa-context/rdfa-1.1

And how close should it align with the in-practice gospel of top stackoverflow answer for "dc vs dc11 vs dcterms" search?
https://stackoverflow.com/questions/47519315/what-is-the-difference-between-dublin-core-terms-and-dublin-core-elements-vocabu

Would be nice if DCMI had an official recommendation that everyone could follow to avoid everyone always being confused by this.  I still don't understand. ;-) Which is less confusing, not including dc and having people define it themselves in potential incompatible ways?  Or defining it as something that may be incompatible with other uses?  Hard to win this battle.

**Comment by gkellogg** [comment]
> @pchampin I have sympathy with what you say... Another possibility that we try to contact Tom Baker and ask him for the 'official' standpoint of DCMI.

IIRC, we contacted Tom for. The RDFa context, and he suggested “dc”=dcmiterms. But, it was controversial still, and I think he later recanted.

Probably best leave it out entirely and use terms like “dct” and “dc11”.

**Comment by azaroth42** [comment]
Agreed with pchampin and gkellogg, so propose dc and dcterms as the prefixes
